tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/deep-config/test-suite-a/testclass-class/testclassgetterproperty-property.html

```diff
@@ -13,7 +13,7 @@ <h1>testClassGetterProperty</h1>
         <p>Test class property with both a getter and a setter.</p>
       </section>
       <section>
-        <h2 id="testclassgetterproperty-signature">Signature</h2><code>/** @virtual */<br>get testClassGetterProperty(): number;<br><br><br>set testClassGetterProperty(newValue: number);</code>
+        <h2 id="testclassgetterproperty-signature">Signature</h2><code>/** @virtual */<br>get testClassGetterProperty(): number;<br><br>set testClassGetterProperty(newValue: number);</code>
```

Comment:
Note for reviewers: this is actually correct. The previous rendering was incorrect. The JSON that API-Extractor generated, which this is derived from, is:
```json
"excerptTokens": [
	{
		"kind": "Content",
		"text": "get testClassGetterProperty(): "
	},
	{
		"kind": "Content",
		"text": "number"
	},
	{
		"kind": "Content",
		"text": ";\n\nset testClassGetterProperty(newValue: number);"
	}
],
```

Reply:
Note that the third entry has _2_ line breaks between the getter and setter excerpts. A bug in `createNodesFromPlainText` was causing extra line breaks to be injected when splitting strings into plain text and line break nodes.